facia/app/controllers/FaciaController.scala

```diff
@@ -262,6 +297,12 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
       })
     }.getOrElse(successful(None))
 
+  private def getSomeCollections(path: String, num: Int, offset: Int = 0): Future[Option[List[PressedCollection]]] =
+      frontJsonFapi.get(path).map(_.flatMap{ faciaPage =>
+        // To-do: change the filter to only exclude thrashers and empty collections, not items such as the big picture
+        Some(faciaPage.collections.filterNot(collection => (collection.curated ++ collection.backfill).length < 2 || collection.displayName == "most popular").drop(offset).take(num))
```

Comment:
might be possible to improve the formatting here a bit, splitting each conceptual step onto a separate line